apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -34,6 +39,104 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
       .then(data => setCourseOfferings(data));
   }, []);
 
+  useEffect(() => {
+    if (!courseOfferings) return;
+    if (!isNewSection) {
+      //  TO DO: refactor for efficiency.  Consider using a flatten-like function (maybe in a helper file?)
```

Comment:
Thanks for adding this comment! This could be included explicitly in a followup ticket that you or I (or even someone else on the team) could pick up soon

Reply:
Created in [TEACH-438](https://codedotorg.atlassian.net/jira/software/projects/TEACH/boards/65?selectedIssue=TEACH-438)

Reply:
Added!